README.rst

```diff
@@ -39,6 +39,8 @@ but use ``nengo_ocl.Simulator`` when creating a simulator for your model::
     plt.plot(sim.trange(), sim.data[probe_b])
     plt.show()
 
+If you are running within ``nengo_gui`` make sure the ``PYOPENCL_CTX`` environment variable has been set. If this variable is not set it will open an interactive prompt which will cause ``nengo_gui`` to get stuck during build.
```

Comment:
Can you add some newlines here so no line is longer than 80-ish characters, to be consistent with the rest of the file?